python3/vim_pandoc/command.py

```diff
@@ -165,6 +165,7 @@ def execute(self, should_open):
                             " {'on_exit': 'pandoc#command#JobHandler'," + \
                             "'on_stdout': 'pandoc#command#JobHandler'," + \
                             "'on_stderr': 'pandoc#command#JobHandler'}))")
+                vim.command("put='Running pandoc...\r'")
```

Comment:
Is there a reason to use a linefeed `\r` as opposed to the more typical newline `\n` here?